addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -65,6 +65,16 @@ class ComposerTextInput extends Component {
         return this.env.models['mail.composer'].get(this.props.composerLocalId);
     }
 
+    /**
+     * @returns {mail.composer}
+     */
+    get textInputPlaceholder() {
+        if (this.composer && this.composer.isLog) {
```

Comment:
Condition on `this.composer` is technical and we would like to get rid of it ideally, so it would be better to do a preventive return empty string for it.